packages/vite/rollup.config.js

```diff
@@ -40,14 +40,16 @@ const clientConfig = {
   input: path.resolve(__dirname, 'src/client/client.ts'),
   external: ['./env', '@vite/env'],
   plugins: [
+    nodeResolve(),
```

Comment:
Opening the door for dependencies (bundling) in client.mjs feels a bit risky.
Simply bundling source-map makes client.mjs grow from 17k to 127k.